app/src/main/java/com/nmakademija/nmaakademija/entity/UserRoles.java

```diff
@@ -6,9 +6,9 @@
  * Created by dekedro on 16.8.24.
  */
 public enum UserRoles {
-    @SerializedName("teacher") TEACHER,
```

Comment:
Most of the time we write annotations over variable such as:
```java
    @SerializedName("teacher")
    TEACHER,

    @SerializedName("member")
    MEMBER,

    @SerializedName("guest")
    GUEST,

    @SerializedName("supervisor")
    SUPERVISOR,

    @SerializedName("administrator")
    ADMINISTRATOR
```

Although this syntax is also correct. I am not sure about space removal. I think `Android studio` wanted to move `MEMBER`, `GUEST` to separate line.